src/osgPlugins/OpenFlight/ReaderWriterATTR.cpp

```diff
@@ -185,7 +185,7 @@ ReaderWriter::ReadResult ReaderWriterATTR::readObject(const std::string& file, c
 
 
 ReaderWriter::WriteResult
-ReaderWriterATTR::writeObject(const osg::Object& object, const std::string& fileName, const Options* /*options*/) const
+ReaderWriterATTR::writeObject(const osg::Object& object, const std::string& fileName, const Options* options) const
```

Comment:
we can probably use the 3.6 version of this file.

Reply:
Reverted changes to this file (pending)